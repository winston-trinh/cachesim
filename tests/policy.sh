./csim -S 2 -K 2 -B 2 -p LRU -t traces/simple_policy.trace
./csim -S 2 -K 2 -B 2 -p FIFO -t traces/simple_policy.trace
./csim -S 2 -K 2 -B 1 -p LRU -t traces/simple_policy.trace
./csim -S 2 -K 2 -B 1 -p FIFO -t traces/simple_policy.trace
./csim -S 16 -K 2 -B 16 -p LRU -t traces/yi.trace
./csim -S 4 -K 2 -B 8 -p LRU -t traces/trans_1.trace
./csim -S 4 -K 4 -B 8 -p LRU -t traces/trans_1.trace
./csim -S 16 -K 2 -B 16 -p LRU -t traces/fifo_s1.trace
./csim -S 16 -K 2 -B 16 -p LRU -t traces/fifo_s2.trace
./csim -S 16 -K 4 -B 16 -p LRU -t traces/fifo_s3.trace
./csim -S 32 -K 2 -B 4 -p LRU -t traces/fifo_m1_1.trace
./csim -S 8 -K 4 -B 4 -p LRU -t traces/fifo_m1_1.trace
./csim -S 32 -K 2 -B 4 -p LRU -t traces/fifo_m2_1.trace
./csim -S 8 -K 4 -B 4 -p LRU -t traces/fifo_m2_1.trace
./csim -S 16 -K 2 -B 16 -p LRU -t traces/fifo_l_1.trace
./csim -S 16 -K 2 -B 16 -p FIFO -t traces/yi.trace
./csim -S 4 -K 2 -B 8 -p FIFO -t traces/trans_1.trace
./csim -S 4 -K 4 -B 8 -p FIFO -t traces/trans_1.trace
./csim -S 16 -K 2 -B 16 -p FIFO -t traces/fifo_s1.trace
./csim -S 16 -K 2 -B 16 -p FIFO -t traces/fifo_s2.trace
./csim -S 16 -K 4 -B 16 -p FIFO -t traces/fifo_s3.trace
./csim -S 32 -K 2 -B 4 -p FIFO -t traces/fifo_m1_1.trace
./csim -S 8 -K 4 -B 4 -p FIFO -t traces/fifo_m1_1.trace
./csim -S 32 -K 2 -B 4 -p FIFO -t traces/fifo_m2_1.trace
./csim -S 8 -K 4 -B 4 -p FIFO -t traces/fifo_m2_1.trace
./csim -S 16 -K 2 -B 16 -p FIFO -t traces/fifo_l_1.trace
