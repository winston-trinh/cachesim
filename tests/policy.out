hits:3 misses:4 evictions:1
hits:2 misses:5 evictions:2
hits:2 misses:5 evictions:2
hits:1 misses:6 evictions:3
hits:4 misses:5 evictions:2
hits:201 misses:37 evictions:29
hits:212 misses:26 evictions:10
hits:8 misses:4 evictions:1
hits:11 misses:7 evictions:3
hits:9 misses:8 evictions:4
hits:61 misses:352 evictions:296
hits:52 misses:361 evictions:329
hits:194 misses:185 evictions:139
hits:171 misses:208 evictions:177
hits:265687 misses:26015 evictions:25983
hits:4 misses:5 evictions:2
hits:192 misses:46 evictions:38
hits:208 misses:30 evictions:14
hits:7 misses:5 evictions:2
hits:11 misses:7 evictions:3
hits:6 misses:11 evictions:7
hits:59 misses:354 evictions:298
hits:51 misses:362 evictions:330
hits:191 misses:188 evictions:142
hits:164 misses:215 evictions:184
hits:263447 misses:28255 evictions:28223
